### ISSUE TITLE
[Feature Request] Allow running wine executables with extra environment variables or through `env`

### ISSUE THREAD

**Issue by wyphan** [brezerk/q4wine#192]
It would be nice if q4wine allows setting extra environment variables to be passed on to the `wine` command. For my use case, I need it to pass `MESA_GL_VERSION_OVERRIDE=4.5` to fix the [GLXBadFBConfig error](https://askubuntu.com/questions/1369900/x-error-of-failed-request-glxbadfbconfig) that I get with some games.

I've tried using `export MESA_GL_VERSION_OVERRIDE=4.5` as the pre run script, but it didn't work.

For now, the only way for me to launch the game is right-clicking on the icon in q4wine, selecting "Console" > "Open console in application directory", then manually type `export MESA_GL_VERSION_OVERRIDE=4.5` and then `wine game.exe` to launch the game.

**Comment by brezerk** [comment]
hi @wyphan thank you for the feature request.

q4wine already allows you to supply extra environment variables.
You can do this using prefix execution template string: 

![зображення](https://user-images.githubusercontent.com/23820/153500005-4cb14109-573a-4451-bbc9-84bbc4aa8773.png)

See description at q4wine home page: https://q4wine.brezblock.org.ua/documentation/en_us/10-prefix-settings.html#advanced

Let me know if is what you are looking for.

Thanks

**Comment by wyphan** [comment]
@brezerk Yes, that's what I'm looking for! It would be nice though if there is some mechanism in q4wine to modify `%ENV_ARGS%`, instead of manually editing the execution template string.
